### ISSUE TITLE
Missing tailwind dependency

### ISSUE THREAD

**Issue by michxymi** [haydenbleasel/next-forge#176]
Hello,

I've trying navigating to the web page or the app page in localhost 3000 and 3001 respectively but I always seem to get the following error:

```
/0.pack.gz_' -> '/home/michxymi/Repos/github.com/trustflare/apps/app/.next/cache/webpack/client-development-fallback/0.pack.gz'
app:dev:  ⨯ ../../node_modules/.pnpm/geist@1.3.1_next@15.0.2_@opentelemetry+api@1.9.0_react-dom@18.3.1_react@18.3.1__react@18.3.1_/node_modules/geist/dist/mono.js
app:dev: An error occurred in `next/font`.
app:dev: 
app:dev: Error: Cannot find module 'tailwindcss'

```

I'm not entirely sure if this is an error of my setup or an actual bug. I did follow the install instructions but still have to define certain env vars. Not sure if that's the problem. It would be nice to be a bit more explicit in the env vars section documentation by adding all the urls of the applications used in the template so users that aren't super familiar with the tooling can easily set the template up. Amazing job by the way, this is a fantastic template.

**Comment by haydenbleasel** [comment]
I just released a big improvement to the env vars a few moment ago. If you can, pull a fresh copy and check out https://www.next-forge.com/docs/env

**Comment by michxymi** [comment]
Env var wise it looks much better now with zog validation and clearer errors.

`api:dev: [stripe] > stripe listen --forward-to localhost:3002/webhooks/stripe
api:dev: [stripe]
app:dev: ❌ Invalid environment variables: {
app:dev:   CLERK_SECRET_KEY: [
app:dev:     'String must contain at least 1 character(s)',
app:dev:     'Invalid input: must start with "sk_"'
app:dev:   ],
app:dev:   CLERK_WEBHOOK_SECRET: [
app:dev:     'String must contain at least 1 character(s)',
app:dev:     'Invalid input: must start with "whsec_"'
app:dev:   ],
app:dev:   RESEND_AUDIENCE_ID: [ 'String must contain at least 1 character(s)' ],
app:dev:   RESEND_FROM: [ 'String must contain at least 1 character(s)', 'Invalid email' ],
app:dev:   DATABASE_URL: [ 'String must contain at least 1 character(s)', 'Invalid url' ],
app:dev:   RESEND_TOKEN: [
app:dev:     'String must contain at least 1 character(s)',
app:dev:     'Invalid input: must start with "re_"'
app:dev:   ],
app:dev:   STRIPE_SECRET_KEY: [
app:dev:     'String must contain at least 1 character(s)',
app:dev:     'Invalid input: must start with "sk_"'
app:dev:   ],
app:dev:   STRIPE_WEBHOOK_SECRET: [
app:dev:     'String must contain at least 1 character(s)',
app:dev:     'Invalid input: must start with "whsec_"'
app:dev:   ],
app:dev:   BETTERSTACK_API_KEY: [ 'String must contain at least 1 character(s)' ],
app:dev:   BETTERSTACK_URL: [ 'String must contain at least 1 character(s)', 'Invalid url' ],
app:dev:   NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY: [
app:dev:     'String must contain at least 1 character(s)',
app:dev:     'Invalid input: must start with "pk_"'
app:dev:   ],
app:dev:   NEXT_PUBLIC_GA_MEASUREMENT_ID: [
app:dev:     'String must contain at least 1 character(s)',
app:dev:     'Invalid input: must start with "G-"'
app:dev:   ],
app:dev:   NEXT_PUBLIC_POSTHOG_KEY: [
app:dev:     'String must contain at least 1 character(s)',
app:dev:     'Invalid input: must start with "phc_"'
app:dev:   ],
app:dev:   NEXT_PUBLIC_POSTHOG_HOST: [ 'String must contain at least 1 character(s)', 'Invalid url' ]
app:dev: }`

Hopefully the tailwind issue will go away once I setup all the required env vars.

**Comment by haydenbleasel** [comment]
Let's hope so! Feel free to use this mock env file and let me know if the Tailwind issue persists.

```
# Server
CLERK_SECRET_KEY="sk_test_JA=="
CLERK_WEBHOOK_SECRET="whsec_test"
RESEND_AUDIENCE_ID="test"
RESEND_FROM="test@test.com"
DATABASE_URL="postgresql://test:test@localhost:5432/test"
RESEND_TOKEN="re_test"
STRIPE_SECRET_KEY="sk_test"
STRIPE_WEBHOOK_SECRET="whsec_test"
BETTERSTACK_API_KEY="test"
BETTERSTACK_URL="https://test.com"

# Client
NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY="pk_test_JA=="
NEXT_PUBLIC_CLERK_SIGN_IN_URL="/sign-in"
NEXT_PUBLIC_CLERK_SIGN_UP_URL="/sign-up"
NEXT_PUBLIC_CLERK_AFTER_SIGN_IN_URL="/"
NEXT_PUBLIC_CLERK_AFTER_SIGN_UP_URL="/"
NEXT_PUBLIC_GA_MEASUREMENT_ID="G-test"
NEXT_PUBLIC_POSTHOG_KEY="phc_test"
NEXT_PUBLIC_POSTHOG_HOST="https://test.com"
NEXT_PUBLIC_APP_URL="http://localhost:3000"
NEXT_PUBLIC_WEB_URL="http://localhost:3001"
NEXT_PUBLIC_DOCS_URL="http://localhost:3004"
NEXT_PUBLIC_VERCEL_PROJECT_PRODUCTION_URL="http://localhost:3002"
```

**Comment by haydenbleasel** [comment]
@michxymi We fixed the env var problem in #180, can you re-init and verify?

**Comment by michxymi** [comment]
> @michxymi We fixed the env var problem in #180, can you re-init and verify?

Last time I tried, env vars were working nicely (or at least they were telling me about the ones I missed). It did fail though with something similar to #190. I've never seen the original tailwind problem again so I think that was some kind of transitive issue that has gone so I'm happy to close this.
